open_seq2seq/parts/cnns/conv_blocks.py

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2018 NVIDIA Corporation
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from six.moves import range
+
+import tensorflow as tf
+
+def conv_bn_actv(layer, name, inputs, filters, kernel_size, activation_fn, strides,
```

Comment:
Can you add a call to this function in ds2_encoder and remove the same function from there? Make sure to run tests to see that everything is working after that

Reply:
There is a slight difference between this function and the one in ds2_encoder ('conv_bn_actv' vs 'conv2d_bn_actv'). Do you want me to update all the configs as well for ds2 and test if that works?